fearless/Modules/NFT/NftDetails/NftDetailsPresenter.swift

```diff
@@ -107,6 +117,10 @@ extension NftDetailsPresenter: NftDetailsViewOutput {
         interactor.setup(with: self)
     }
 
+    func viewAppeared() {
+        interactor.initialSetup()
```

Comment:
it is not initial setup, cause init setup is func setup(with output: NftDetailsInteractorOutput), perhaps viewApeared also?